script.myepisodes/default.py

```diff
@@ -14,6 +14,7 @@
 from myepisodes import MyEpisodes
 
 _addon = xbmcaddon.Addon()
+_kodiversion = float(_addon.getAddonInfo('version')[0:4])
```

Comment:
This isn't the Kodi version, it's the version of your addon.
```suggestion
_kodiversion = float(xbmcaddon.Addon('xbmc.addon')..getAddonInfo('version')[0:4])
```